docs/wiki/Getting started - Setup environment.md

```diff
@@ -2,12 +2,21 @@ The repository is built so that you can create your own 1:1 instance and be able
 
 This requires several steps:
 
-1. [Configure your Azure environment](#1-configure-your-azure-environment)
-1. [Fork/clone the repository into your DevOps environment](#2-forkclone-the-repository-into-your-devops-environment)
-1. [Configure the CI environment](#3-configure-the-ci-environment)
-1. [Deploy dependencies](#4-deploy-dependencies)
-1. [Update module parameter files](#5-update-module-parameter-files)
-1. [(Optional) Convert library to ARM](#6-optional-convert-library-to-arm)
+- [1. Configure your Azure environment](#1-configure-your-azure-environment)
+- [2. Fork/clone the repository into your DevOps environment](#2-forkclone-the-repository-into-your-devops-environment)
```

Comment:
Something tells me you're using a markdown extension. If you can, please undo the number updates. It better to have markdown take care of it using native capabilities

Reply:
Not sure though if it works with nested lists

Reply:
indeed. should now be removed, thanks!